tools/mcp/daemon/zzz_od_daemon.py

```
@@ -0,0 +1,221 @@
+        启动结果信息。
+    """
+    existing_proc = find_zzz_od_mcp_server_process()
+    if existing_proc:
+        return f"[OK] 主 MCP server 已在运行 (PID: {existing_proc.pid})"
+
+    if is_port_in_use(port):
+        return f"[WARN] 端口 {port} 已被占用，可能有其他程序在使用"
+
+    try:
+        cmd = f'cd /d "{PROJECT_ROOT}" && uv run --env-file .env python -m zzz_od.backend.entry.server --port {port}'
+        process = subprocess.Popen(
```

Ruff (0.15.20)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)